third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2211,6 +2255,17 @@ func expandVerticalPodAutoscaling(configured interface{}) *containerBeta.Vertica
 		Enabled: config["enabled"].(bool),
 	}
 }
+
+func expandWorkloadIdentityConfig(configured interface{}) *containerBeta.WorkloadIdentityConfig {
+	l := configured.([]interface{})
+	if len(l) == 0 {
+		return nil
+	}
+	config := l[0].(map[string]interface{})
```

Comment:
This line panics when the value for `identity_namespace` is unset, as triggered by your test. Setting it to `Required` will resolve this.